INTEGRATION_TESTS.md

```diff
@@ -119,6 +119,9 @@ This will:
 Now when you run `npm test` it will synth the integ app and compare the result with the snapshot.
 If the snapshot has changed the same process must be followed to update the snapshot.
 
+*Note - When running integration tests, refrain from defining the `env.account` property for stacks. This ensures that the synthesized
```

Comment:
I don't think this is the answer to this issue. I don't think account numbers are inherently considered sensitive information, but even if we decide to them as such, we should write a linter rule that explicitly prevents users from including them in snapshots. 

This might not apply to all tests, some of them need real account numbers (like cross account tests)

Reply:
@comcalvi Agree. I have rephrased it this way

*Note - When running integration tests, `env.account` is generally not required unless some special cases like cross-account tests.*

Feel free to suggest changes. I think we still have to remind users that `env.account` is generally not required and not recommended. Thoughts?